doc/using/as-a-function.chapter.md

```diff
@@ -0,0 +1,147 @@
+# Nixpkgs as a Function {#sec-nixpkgs-function}
+
+Depending on how you use Nixpkgs, you may interact with it as a [Nix function](https://nix.dev/manual/nix/2.23/language/constructs.html#functions).
```

Comment:
Is it intended that this manual link directly targets a version whilst all others target `latest`? That kind of looks like being unintentional on first view.

Reply:
Good catch!
I've [changed](https://github.com/NixOS/nixpkgs/compare/de0c0b6823fecca851d11555faa82b30b9e93225..d159fca38deef57f11abef904d6e0960982666e2) it to `latest` and gathered up the remaining URLs.